src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CONS = "supply";
+    private static final String BUY_CONS = "buy";
+    private static final String RESULT_CONS = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals = readFile(fromFileName);
+        int totalSupply = totals[0];
+        int totalBuy = totals[1];
+        writeToFile(toFileName, totalSupply, totalBuy);
+    }
+
+    public int[] readFile(String fromFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] values = line.split(",");
+                String type = values[0];
+                int amount = Integer.parseInt(values[1]);
+                if (type.equals(SUPPLY_CONS)) {
+                    totalSupply += amount;
+                } else if (type.equals(BUY_CONS)) {
+                    totalBuy += amount;
+                }
```

Comment:
The loop inside `readFile` method could be refactored into a separate method to improve readability and maintainability.